services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/student_overview_table.jsx

```diff
@@ -82,7 +83,13 @@ export default class extends React.Component {
           <a className={scoreInfo.linkColor} href={`/activity_sessions/anonymous?activity_id=${row.activity_id}`}>{row.name}</a>
         </td>
         <td>{this.completedStatus(row)}</td>
-        <td className={`score ${blurIfNotPremium}`}>{scoreInfo.content}</td>
+        { scoreInfo.tooltip ?
```

Comment:
Two nits:
- an if/else is much easier to read in this context than a ternary operator
- While `scoreInfo.tooltip` works as a conditional currently, what we really want to condition on is 'is this type of activity gradeable'. In the future, we may want tooltips for graded activities, for example. Consider making the intent more obvious by: `const notGraded = scoreInfo.context` or some other strategy

Reply:
Thanks for the notes! (Second one was really astute, I hadn't thought of that case). I incorporated the changes you suggested here.